.github/workflows/deploy-wallet-at-pr.yml

```diff
@@ -125,55 +88,28 @@ jobs:
       - name: Install protoc
         run: sudo apt-get install -y protobuf-compiler
 
-      - name: Install wasm-pack
-        uses: jetli/wasm-pack-action@v0.4.0
-        with:
-          version: "v0.10.3"
-
-      - name: Rustup add target
-        run: rustup target add wasm32-unknown-unknown
-
       - name: build the site
         working-directory: ./apps/namadillo
         run: yarn build
         env:
           NAMADA_INTERFACE_NAMADA_ALIAS: "Namada Devnet"
           NAMADA_INTERFACE_NAMADA_CHAIN_ID: "internal-devnet-6be.86067e06a5"
           NAMADA_INTERFACE_NAMADA_URL: "https://proxy.heliax.click/internal-devnet-6be.86067e06a5"
-      - name: Deploy to Netlify
```

Comment:
out of curiosity, is there a reason to not deploy on this case?

Reply:
It's basically just not being used right now. I wasn't sure but I checked with Justin a couple of weeks ago.